src/main/java/com/glencoesoftware/ldaptool/Main.java

```diff
@@ -20,101 +20,84 @@
 
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
-import org.springframework.ldap.core.DistinguishedName;
 import org.springframework.ldap.core.LdapTemplate;
 
 import ch.qos.logback.classic.Level;
 import ome.logic.LdapImpl;
-import ome.logic.LdapImpl.GroupLoader;
-import ome.model.meta.Experimenter;
 import ome.system.OmeroContext;
 
 import java.io.File;
 import java.io.FileInputStream;
-import java.lang.reflect.Constructor;
-import java.lang.reflect.Field;
-import java.util.Arrays;
-import java.util.List;
 import java.util.Properties;
-import java.util.StringJoiner;
-import java.util.concurrent.Callable;
+import java.util.concurrent.atomic.AtomicBoolean;
 
 import picocli.CommandLine;
-import picocli.CommandLine.ArgGroup;
+import picocli.CommandLine.Command;
 import picocli.CommandLine.Option;
-import picocli.CommandLine.Parameters;
+import picocli.CommandLine.ParseResult;
 
 /**
  * @author Chris Allan <callan@glencoesoftware.com>
  */
-public class Main implements Callable<Integer>
+@Command(
+    mixinStandardHelpOptions = true,
+    subcommands = {Password.class, Search.class},
+    exitCodeOnExecutionException = 100
+)
+public class Main
 {
     private static final Logger log =
             LoggerFactory.getLogger(Main.class);
 
     @Option(
-        names = "--help",
-        usageHelp = true,
-        description = "Display this help and exit"
+        names = "--log-level",
+        description = "Change logging level; valid values are " +
+            "OFF, ERROR, WARN, INFO, DEBUG, TRACE and ALL. " +
+            "(default: ${DEFAULT-VALUE})"
```

Comment:
Proposing to add 
```suggestion
            "(default: ${DEFAULT-VALUE})",
        scope = ScopeType.INHERIT
```

so that options can be inherited by all subcommands